debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -472,10 +441,6 @@ public OracleVersion getOracleVersion() {
         return oracleVersion;
     }
 
-    public Tables.ColumnNameFilter getColumnFilter() {
```

Comment:
How does the column filter work then now? Is it inherited from the framework? Is there a test for the Oracle connector for this?

Reply:
Yes the filter is part of the common framework; however I do need to add a test for this.  I thought this was tested in the core repository but looking at the connectors there they each have at least one test for this so I'll add one here for consistency.